apps/prairielearn/src/ee/pages/instructorAiGenerateJobs/instructorAiGenerateJobs.sql

```diff
@@ -3,7 +3,7 @@ SELECT
   to_jsonb(j.*) AS job,
   to_jsonb(u.*) AS user
 FROM
-  jobs AS j
+  job_sequences AS j
```

Comment:
right now, not doing this in `instructorAiGenerateJob.sql` as well because i'd need to call `selectJobsByJobSequenceId`, to extract data, which uses a similar query. LMK if that would be preferable.